roles/openshift_health_checker/openshift_checks/package_availability.py

```diff
@@ -36,13 +36,11 @@ def master_packages(rpm_prefix):
         return [
```

Comment:
This is such a random cross section of packages, I think this needs to be cleaned up at some point.

Reply:
I think it would be safe to assume that if `-clients` RPM is available so is the rest